.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,101 @@ assignees: ''
+- [ ] Added the onboarded employee's UCSC account to the `firecloud-cgl` group in Terra
+- [ ] Assigned this ticket to the onboarded employee
 
-- [ ] @hannes-ucsc: [GCP Credentials](https://console.cloud.google.com/)
+## Onboarded employee actions
 
-- [ ] @theathorn, Erich Weiler: Membership in the `firecloud-cgl` Terra group
+- [ ] Sent a request email to the system administrator (cc: hiring supervisor) asking them to:
+    - Create for you a user account on the dev GitLab instances
+    - Assign the `developer` role to the account
+- [ ] Assigned this ticket to the hiring supervisor
+
+## Hiring supervisor actions
+
+- [ ] Approved (by replying to) the request email sent by the onboarded employee to the system administrator
+- [ ] Assigned this ticket to the system administrator
+
+## System administrator actions
+
+- [ ] Verified the authenticity of the approval
+- [ ] Verified the GitHub user account is associated with the onboarded employee's UCSC account email
+- [ ] Created a GitLab user account for the onboarded employee on the `dev` instance of GitLab
+- [ ] Created a GitLab user account for the onboarded employee on the `anvildev` instance of GitLab
+- [ ] Added the onboarded employee to exactly one of the groups on the `dev` instance of GitLab and set the `Max Role` to `Developer`
+- [ ] Added the onboarded employee to exactly one of the groups on the `anvildev` instance of GitLab and set the `Max Role` to `developer`
+- [ ] Associated the new GitLab user accounts with the onboarded employee's GitHub user account
+- [ ] Added the onboarded employee to the `Azul Developers` GitHub team
+- [ ] Assigned the `developer` role to the onboarded employee's GitLab user accounts.
+- [ ] Added the onboarded employee's UCSC account to the `platform-hca-dev` Google project and assigned the `developer` role
+- [ ] Added the onboarded employee's UCSC account to the `platform-anvil-dev` Google project and assigned the `developer` role
+- [ ] Added the onboarded employee's Google Workspace account to the `platform-hca-dev` Google project and assigned `Project Editor` permissions
+- [ ] Added the onboarded employee's Google Workspace account to the `platform-anvil-dev` Google project and assigned `Project Editor` permissions
+- [ ] Assigned this ticket to the project manager
+
+## Project manager actions
+
+- [ ] Added the onboarded employee to the [UCSC-CGL Slack](https://ucsc-cgl.slack.com)
+- [ ] Assigned this ticket to the system administrator
+
+## System administrator actions
+
+- [ ] Gave the onboarded employee push access to the Azul Repository on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+- [ ] Gave the onboarded developer access to development CI/CD systems on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+- [ ] Gave the onboarded employee [GCP Credentials](https://console.cloud.google.com/)
+- [ ] Added the onboarded employee's Google Workspace account to the `azul-dev` group in Terra
```

Comment:
Add checkbox for `azul-anvil-dev`
Add checkbox for `azul-anvil-prod` (requires managed access certification)